queries.sql

```diff
@@ -22,4 +22,36 @@ SELECT * FROM ANIMALS WHERE NEUTERED IS TRUE;
 SELECT * FROM ANIMALS WHERE NAME NOT LIKE 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-SELECT * FROM ANIMALS WHERE WEIGHT_KG >= 10.4 AND WEIGHT_KG <= 17.3;
\ No newline at end of file
+SELECT * FROM ANIMALS WHERE WEIGHT_KG >= 10.4 AND WEIGHT_KG <= 17.3;
+
+BEGIN;
+    UPDATE ANIMALS SET SPECIES = 'digimon' WHERE NAME LIKE '%mon';
+    UPDATE ANIMALS SET SPECIES = 'pokemon' WHERE NAME NOT LIKE '%mon';
+COMMIT;
+
```

Comment:
- [ ]  I think you forget to add this query here to delete all records in the animals table, then roll back the transaction. so kindly add it since it is one of the requirements ✅♻✨💯

![image](https://user-images.githubusercontent.com/58553711/208214416-04fa233b-dc92-4445-9fb2-f4f091751e1e.png)

Reply:
Actually, I didn't leave it out. If you check the screenshots you'll notice it is there

Reply:
- I checked but I mean you didn't add it in the file ✅